charts/customer-center/README.md

```diff
@@ -0,0 +1,158 @@
+| backend.service.externalPort | int | `3000` | External Port of backend service |
+| backend.service.internalPort | int | `3000` | Internal Port of backend service |
+| backend.service.name | string | `"customer-center-backend"` | Backend service name |
+| backend.service.type | string | `"ClusterIP"` | Backend service type |
+| backend.settings.appHost | string | `"my.example.com"` | Application host name |
+| backend.settings.appName | string | `"Customer Center"` | Application name |
+| backend.settings.auth | object | `{"expirePassword":3600,"roleAdmin":"ldap-role-admin","roleEmployee":"ldap-role-employee","secret":"my-super-secret","usersCustomersGroupSearchBase":"cn=groups,ou=customers,cn=accounts,dc=sso,dc=example,dc=com","usersCustomersGroupSearchFilter":"(&(uniqueMember={{dn}})(objectClass=posixGroup))","usersCustomersSearchBase":"ou=customers,cn=accounts,dc=sso,dc=example,dc=com","usersCustomersSearchFilter":"(uid={{username}})","usersGroupSearchBase":"cn=groups,ou=staff,cn=accounts,dc=sso,dc=example,dc=com","usersGroupSearchFilter":"(&(uniqueMember={{dn}})(objectClass=posixGroup))","usersSearchBase":"ou=staff,cn=accounts,dc=sso,dc=example,dc=com","usersSearchFilter":"(uid={{username}})"}` | Authentication settings |
+| backend.settings.auth.expirePassword | int | `3600` | Timeout in seconds before Redis key expiration |
+| backend.settings.auth.roleAdmin | string | `"ldap-role-admin"` | Admin role to use |
+| backend.settings.auth.roleEmployee | string | `"ldap-role-employee"` | Employee role to use |
+| backend.settings.auth.secret | string | `"my-super-secret"` | Authentication settings |
+| backend.settings.auth.usersCustomersGroupSearchFilter | string | `"(&(uniqueMember={{dn}})(objectClass=posixGroup))"` | Search filter to get customers users groups name |
+| backend.settings.auth.usersCustomersSearchBase | string | `"ou=customers,cn=accounts,dc=sso,dc=example,dc=com"` | Search base to find customers users group |
+| backend.settings.auth.usersCustomersSearchFilter | string | `"(uid={{username}})"` | Search filter to get customers users name |
+| backend.settings.auth.usersGroupSearchBase | string | `"cn=groups,ou=staff,cn=accounts,dc=sso,dc=example,dc=com"` | Search base to find users groups |
+| backend.settings.auth.usersGroupSearchFilter | string | `"(&(uniqueMember={{dn}})(objectClass=posixGroup))"` | Search filter to get users groups name |
+| backend.settings.auth.usersSearchBase | string | `"ou=staff,cn=accounts,dc=sso,dc=example,dc=com"` | Search base to find users |
+| backend.settings.auth.usersSearchFilter | string | `"(uid={{username}})"` | Search filter to get users username |
+| backend.settings.debug | string | `"app:info,app:warn,app:error"` | Verbosity of logging |
+| backend.settings.keycloak | object | `{"clientID":"this-is-my-client-id","clientSecret":"this-is-my-client-secret","host":"https://sso.example.com","tokenPath":"/auth/realms/example/protocol/openid-connect/token"}` | Keycloak settings |
```

Comment:
Please remove the large JSON dump in these docs. Also, the `keycloak` noun might better be replaced with the generic term `idp`.